File: rxjava/src/main/java/com/chiclaim/rxjava/MainActivity.java
package com.chiclaim.rxjava;

import android.os.Bundle;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

/**
 * Created by chiclaim on 2016/03/23
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.fragment_container);
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(false);
        }

        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
        ft.add(R.id.container, new MainFragment()).commit();
    }


}

File: android-base/ProguardDemo/app/src/main/java/com/chiclaim/proguarddemo/MainActivity.kt
package com.chiclaim.proguarddemo

import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val imageResId: Int = resources.getIdentifier("ic_get_by_identifier", "drawable", "com.chiclaim.proguarddemo")
        val image = findViewById<ImageView>(R.id.image_view)
        image.setImageResource(imageResId)
    }
}
